src/components/main-panel/footerData.ts

```diff
@@ -0,0 +1,24 @@
+export const footerData = [
+  {
+    heading: "Company",
+    data: ["About", "Jobs", "For the Record"],
+  },
+  {
+    heading: "Communities",
+    data: ["For Artists", "Developers", "Advertising", "Investors", "Vendors"],
+  },
+  {
+    heading: "Useful links",
+    data: ["Support", "Free Mobile App"],
+  },
+  {
+    heading: "Spotify Plans",
+    data: [
+      "Premium Individual",
+      "Premium Duo",
+      "Premium Family",
+      "Premium Student",
+      "Spotify Free",
+    ],
+  },
+];
```

Comment:
should be moved to the same directory called `data`

Reply:
Got it.